### ISSUE TITLE
Feature Request: Make a method to list all WiFi SSIDs

### ISSUE THREAD

**Issue by pedrosimao** [tadasr/react-native-iot-wifi#5]
The idea is to implement a method that would list all available WiFi SSIDs.
Is it possible? Complex to implement?

**Comment by tadasr** [comment]
Yes, but it will work only on Android, iOS API does not provide wifi scan results.

**Comment by notjulian** [comment]
@pedrosimao this one has that feature
https://github.com/JuanSeBestia/react-native-wifi-reborn
